xbmc/SortFileItem.h

```diff
@@ -39,6 +39,7 @@ typedef enum {
   SORT_METHOD_COUNTRY,
   SORT_METHOD_YEAR,
   SORT_METHOD_VIDEO_RATING,
+  SORT_METHOD_VIDEO_USER_RATING,
```

Comment:
please also add to
https://github.com/xbmc/xbmc/blob/master/xbmc/interfaces/legacy/ModuleXbmcplugin.h#L193